modules/serverless-creating-broker.adoc

```diff
@@ -3,41 +3,144 @@
 // * serverless/knative_eventing/serverless-using-brokers.adoc
 
 [id="serverless-creating-broker_{context}"]
-= Creating a broker manually
+= Creating a Broker as a developer
 
-To create a broker, you must create a `ServiceAccount` for each namespace and give that `ServiceAccount` the required RBAC permissions.
+. Optional. If you do not have an existing event sink created to consume events from the Broker, you can create a simple Knative Service as an event sink.
+.. Copy the following into a YAML file:
++
 
-.Prerequisites
-* Knative Eventing installed, which includes the `ClusterRole`.
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: default
+spec:
+  template:
+    spec:
+      containers:
+        - image: quay.io/openshift-knative/knative-eventing-sources-event-display
+----
 
-.Procedure
-. Create the `ServiceAccount` objects:
+.. Create the Service by applying the YAML file:
 +
+
+[source,terminal]
 ----
-$ oc -n default create serviceaccount eventing-broker-ingress
-$ oc -n default create serviceaccount eventing-broker-filter
+$ oc apply -f <FILENAME>
 ----
 
-. Give those objects RBAC permissions:
+. Optional. Create an event source that emits events to the `default` Broker. The following example shows how you can create a basic PingSource.
+.. Copy the following into a YAML file:
 +
+
+[source,yml]
 ----
-$ oc -n default create rolebinding eventing-broker-ingress \
-  --clusterrole=eventing-broker-ingress \
-  --serviceaccount=default:eventing-broker-ingress
-$ oc -n default create rolebinding eventing-broker-filter \
-  --clusterrole=eventing-broker-filter \
-  --serviceaccount=default:eventing-broker-filter
+apiVersion: sources.knative.dev/v1alpha2
+kind: PingSource
+metadata:
+  name: ping-source
+spec:
+  schedule: "*/1 * * * *"
+  jsonData: '{"message": "Hello serverless!"}'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1beta1
+      kind: Broker
+      name: default
 ----
 
-. Create the broker:
 +
+..  Create the event source by applying the YAML file:
+
++
+[source,terminal]
+----
+$ oc apply -f <FILENAME>
+----
+
+. Create the Broker. You can do this by creating a Trigger object that uses the `knative-eventing-injection: enabled` annotation.
++
+[IMPORTANT]
+====
+If you create a Broker using the `knative-eventing-injection: enabled` annotation, you cannot delete this Broker without cluster administrator permissions.
+If you delete the Broker without having a cluster administrator remove this annotation first, the Broker will be created again after deletion.
```

Comment:
Is there a way to create a Broker without needing an admin to remove it? If so, I'd publish that method instead. If not, I'd clarify that this is the only way and that you will always need a cluster admin to delete the Broker.